.github/workflows/cron-job-templating/templating-script.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+
+import sys
+from jinja2 import Template
+
+
+def validate_epoch_input(cli_args):
+    try:
+        epoch = cli_args[1]
+    except Exception:
+        raise ValueError('Missing required parameter. Epoch must be provided.')
+
+    return epoch
+
+
+def write_recipe(jinja):
```

Comment:
This function might work just fine elided into the `if __name__ == '__main__':` body, rather than sitting here as a standalone function.